examples/stm32/nucleo-f746zg-freertos-mip/main.c

```diff
@@ -47,37 +68,33 @@ static void server(void *args) {
   // IP configuration. If IP/mask/GW are unset, DHCP is going to be used
   MG_INFO(("Initializing Ethernet driver"));
   ethernet_init();
-  struct mip_driver_stm32_data driver_data = {.mdc_cr = 4};  // See driver_stm32.h
-  struct mip_if mif = {
-      .mac = {2, 0, 1, 2, 3, 5},
```

Comment:
The example code must be optimised for the potential user, not us!
My rationale was to avoid MAC clashes when more that 1 device gets started in one LAN.

Reply:
Good point! I manually change them but I'm not the regular user...